rero_ils/modules/locations/jsonschemas/locations/location-v0.0.1.json

```diff
@@ -112,6 +116,63 @@
           "pattern": "^https://ils.rero.ch/api/libraries/.*?$"
         }
       }
+    },
+    "contact_email": {
+      "title": "Contact email",
+      "type": "string",
+      "format": "email",
+      "pattern": "^\\w+([\\.-]?\\w+)*@\\w+([\\.-]?\\w+)*(\\.\\w{2,})+$",
+      "minLength": 6,
+      "form": {
+        "validation": {
+          "messages": {
+            "pattern": "Email should have at least one <code>@</code> and one <code>.</code>"
+          }
+        }
+      }
+    },
+    "allow_request": {
+      "title": "Allow request",
+      "type": "boolean",
+      "description": "Is request allowed for items linked to this location ?",
```

Comment:
I would not use a question as description. Moreover you have an extra space before `?`

Reply:
update